tests/cli/test_cli.py

```diff
@@ -1372,6 +1372,46 @@ def test_make(ctx):
     _run_command([cl, 'rm', '--force', uuid2])  # force the deletion
     _run_command([cl, 'rm', '-r', uuid1])  # delete things downstream
 
+    # test using make to replicate bundles between storages
+    if os.environ.get("CODALAB_ALWAYS_USE_AZURE_BLOB_BETA") == '1':
+        bundle_store_name = random_name()
+        bundle_store_uuid = _run_command(
+            [
+                cl,
+                "store",
+                "add",
+                "--name",
+                bundle_store_name,
+                '--url',
+                'azfs://devstoreaccount1/bundles',
+            ]
+        )
+        # 1) blob storage -> local filesystem
```

Comment:
what do the 1, 2, 3 refer to? the items in parent_child_store? if so, we might want to move those comments to be inline with the array item declarations